spec/sections/copies.bs

```diff
@@ -165,6 +165,28 @@ dictionary GPUImageCopyTexture {
         Defines which aspects of the {{GPUImageCopyTexture/texture}} to copy to/from.
 </dl>
 
+<div algorithm data-timeline=const>
+    The <dfn abstract-op>texture copy subresource</dfn> for depth slice or array layer |z| of {{GPUImageCopyTexture}}
+    |copyTexture| is determined by running the following steps:
```

Comment:
> One open questions I have: Is there a specific hierarchy to texture layers/mips/aspects? e.g.: Does each mip have layers, or does each layer have mips? Does it matter?

No, I don't think there's a hierarchy to them (though I also don't think it matters).
Every layer has the same mips/aspects,
every mip has the same layers/aspects,
every aspect has the same mips/layers.
So it's a dense 3d array of subresources. (Well, it's never _actually_ 3d because multi-aspect textures can't be mipmapped, but conceptually 3d.)